src/library_noderawfs.js

```diff
@@ -75,16 +75,19 @@ mergeInto(LibraryManager.library, {
         throw new FS.ErrnoError(ERRNO_CODES.EINVAL);
       }
       stream.position = position;
+      stream.seekable = true;
```

Comment:
should this be in `open()`?

Reply:
Hmm, is the issue that the standard streams are created as nodefs?

Reply:
aren't standard streams created in noderawfs's `createStandardStreams()`?

so in `open()` can it always be assumed that the file is seekable since it's not a standard stream or should i check whether it's a regular file with something like `fs.fstatSync().isFile()`?

Reply:
Oh, I forgot node has that `createStandardStreams` method, thanks.

I think the best thing is to create the streams as explicitly non-seekable, and everything else (normal files) as seekable. In more detail, in `createStandardStreams` the streams could have `seekable: false`, and in `open` (which I think happens on the standard streams, and later on?), the code could be
```
  // if marked as non-seekable, leave it that way; otherwise, normal files
  // etc. are all seekable
  if (stream.seekable !== false) {
    stream.seekable = true;
  }
```
```

Reply:
in the new version i set `stream.seekable` to false in `createStandardStreams` and to true in `open` (since it isn't called on standard streams)